diff -ru a/geometry/nokia b/geometry/nokia
--- a/geometry/nokia	2011-11-08 05:21:52.632680002 +1100
+++ b/geometry/nokia	2011-11-08 08:19:32.668680003 +1100
@@ -117,7 +117,7 @@
 
     shape.cornerRadius= 1.5;
     shape "NORM"  { { [18,15] } };
-    shape "SPCE"  { { [36,15] } };
+    shape "SPCE"  { { [54,15] } };
     shape "FRAME" { cornerRadius= 2.5, { [243,54] } };
 
     solid "FRAME" {
@@ -160,3 +160,54 @@
     }; // End of "Keys" section
 
 }; // End of "rx51" geometry
+
+xkb_geometry "nokiarx44" {
+
+    description= "Nokia RX-44";
+    width= 346.5; // 11x8.5mm + 2x11mm border
+    height= 102.0; // 4x7mm + 2x3mm border
+
+    shape.cornerRadius= 1.5;
+    shape "NORM"  { { [25.5,21.0] } };
+    shape "FUNC"  { { [38.25,21.0] } };
+    shape "SPCE"  { { [76.5,21.0] } };
+
+    section.left= 33.0;
+    row.left= 0.0;
+    key.shape= "NORM";
+    key.gap= 0;
+    section "Keys" {
+	top= 0;
+	row {
+	    top= 9.0;
+	    keys { <AD01>, <AD02>, <AD03>, <AD04>, <AD05>,
+		   <AD06>, <AD07>, <AD08>, <AD09>, <AD10>,
+		   <BKSP>
+	    };
+	};
+	row {
+	    top= 30.0;
+	    keys { <AC01>, <AC02>, <AC03>, <AC04>, <AC05>,
+		   <AC06>, <AC07>, <AC08>, <AC09>, <AC11>,
+		   <RTRN>
+	    };
+	};
+	row {
+	    top= 51.0;
+	    keys { <AB01>, <AB02>, <AB03>, <AB04>, <AB05>,
+		   <AB06>, <AB07>, <AC10>, <AE11>, <AE12>,
+		   <RTRN>
+	    };
+	};
+	row {
+	    top= 72.0;
+	    keys { { <FN>, "FUNC", color="grey20" },
+		   <LFSH>, <COMP>,
+		   { <SPCE>, "SPCE", color="grey20" },
+		   <AB08>, <AB09>, <RCTL>,
+                   { <LFSH>, "FUNC", color="grey20" }
+	    };
+	};
+    }; // End of "Keys" section
+
+}; // End of "rx44" geometry
diff -ru a/symbols/nokia_vndr/rx-44 b/symbols/nokia_vndr/rx-44
--- a/symbols/nokia_vndr/rx-44	2009-12-11 11:44:36.000000000 +1100
+++ b/symbols/nokia_vndr/rx-44	2011-11-08 08:16:04.224680002 +1100
@@ -269,7 +269,7 @@
 
     key <COMP>	{	[	Multi_key,		Multi_key,		Multi_key,		Multi_key		]	};
     key <LFSH>	{	[	Shift_L,		Shift_L,		Shift_L,		Shift_L			]	};
-    key <FN>	{	[	ISO_Level3_Shift,	ISO_Level3_Shift,	ISO_Level3_Shift,	ISO_Level3_Shift	]	};
+    key <LALT>	{	[	ISO_Level3_Shift,	ISO_Level3_Shift,	ISO_Level3_Shift,	ISO_Level3_Shift	]	};
     key <SPCE>	{	[	space,			space,			space,			space			]	};
     key <RCTL>	{	[	Control_R,		Control_R,		Control_R,		Control_R		]	};
     key <TAB>	{	[	Tab,			Tab,			Tab,			Tab			]	};
