From c31a1d577b87236a7cd10bbd38d63de143638576 Mon Sep 17 00:00:00 2001
From: Matti Kosola <matti.kosola@jolla.com>
Date: Mon, 2 Mar 2020 14:25:20 +0000
Subject: [PATCH 1/6] build without docs so we don't require xorg macros

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 Makefile.am  |  2 +-
 configure.ac | 11 -----------
 2 files changed, 1 insertion(+), 12 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index e0b3168..2c789d6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 AUTOMAKE_OPTIONS = foreign
 
-SUBDIRS = compat geometry keycodes po rules symbols types docs man
+SUBDIRS = compat geometry keycodes po rules symbols types
 
 pkgconfigdir = $(datadir)/pkgconfig
 pkgconfig_DATA = xkeyboard-config.pc
diff --git a/configure.ac b/configure.ac
index 51f070c..cdbb40a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -3,15 +3,6 @@ AC_CONFIG_SRCDIR(rules/base.xml.in)
 AM_INIT_AUTOMAKE([foreign dist-bzip2])
 AM_MAINTAINER_MODE
 
-# Require X.Org macros 1.12 or later for XORG_WITH_XSLTPROC
-m4_ifndef([XORG_MACROS_VERSION],
-          [m4_fatal([must install xorg-macros 1.12 or later before
-          running autoconf/autogen])])
-XORG_MACROS_VERSION(1.12)
-XORG_MANPAGE_SECTIONS
-XORG_WITH_XSLTPROC
-AC_PROG_SED
-
 AC_SUBST(VERSION)
 
 AC_ARG_WITH( xkb_base,
@@ -103,8 +94,6 @@ symbols/sun_vndr/Makefile
 symbols/xfree68_vndr/Makefile
 types/Makefile
 xkeyboard-config.pc
-docs/Makefile
-man/Makefile
 ])
 
 echo '***********************************************************'
-- 
1.8.3-rc3

