From 2187dda6bbad3b8d608ff1b25cd168c39a94e5dc Mon Sep 17 00:00:00 2001
From: Joona Petrell <joona.petrell@jollamobile.com>
Date: Tue, 3 Oct 2017 17:12:08 +0300
Subject: [PATCH 4/4] [sbj] Map Select key. Contributes to JB#39965

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 keycodes/evdev         | 1 +
 symbols/jolla_vndr/sbj | 1 +
 2 files changed, 2 insertions(+)

diff --git a/keycodes/evdev b/keycodes/evdev
index 6bb9142..0a89b76 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -287,6 +287,7 @@ default xkb_keycodes "evdev" {
 	<I372> = 372;   // #define KEY_FAVORITES           364
 	<I382> = 382;   // #define KEY_KEYBOARD            374
 	<I569> = 569;   // #define KEY_ROTATE_LOCK_TOGGLE  561
+	<I361> = 361;	// #define KEY_SELECT              353
 	<I380> = 380;   // #define KEY_FULL_SCREEN         372
 	<I536> = 536;   // #define KEY_CAMERA_FOCUS        528
 	<I774> = 774;   // #define KEY_CAMERA_SNAPSHOT     766 - not in vanilla kernel
diff --git a/symbols/jolla_vndr/sbj b/symbols/jolla_vndr/sbj
index 6c378eb..dd7f31e 100644
--- a/symbols/jolla_vndr/sbj
+++ b/symbols/jolla_vndr/sbj
@@ -60,4 +60,5 @@ xkb_symbols "common-keys" {
     // These should be camera focus and camera snapshot, but there are no better symbols available
     key <I536>  { [     XF86ContrastAdjust                                              ] };
     key <I774>  { [     XF86WebCam                                                      ] };
+    key <I361>  { [     Select                                                          ] };
 };
-- 
1.8.3-rc3

