From e49a3473d264783b612be17635cfb4427171ce6b Mon Sep 17 00:00:00 2001
From: Joona Petrell <joona.petrell@jollamobile.com>
Date: Tue, 3 Oct 2017 17:12:08 +0300
Subject: [PATCH 6/6] [sbj] Map Select key. Contributes to JB#39965

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 keycodes/evdev         | 1 +
 symbols/jolla_vndr/sbj | 1 +
 2 files changed, 2 insertions(+)

diff --git a/keycodes/evdev b/keycodes/evdev
index a638205..bb0e232 100644
--- a/keycodes/evdev
+++ b/keycodes/evdev
@@ -281,6 +281,7 @@ default xkb_keycodes "evdev" {
 	<I251> = 251;	// #define KEY_BRIGHTNESS_CYCLE    243
 	<I252> = 252;	// #define KEY_BRIGHTNESS_ZERO     244
 	<I253> = 253;	// #define KEY_DISPLAY_OFF         245
+	<I361> = 361;	// #define KEY_SELECT              353
 	<I536> = 536;   // #define KEY_CAMERA_FOCUS        528
 	<I774> = 774;   // #define KEY_CAMERA_SNAPSHOT     766 - not in vanilla kernel
 
diff --git a/symbols/jolla_vndr/sbj b/symbols/jolla_vndr/sbj
index 6c378eb..dd7f31e 100644
--- a/symbols/jolla_vndr/sbj
+++ b/symbols/jolla_vndr/sbj
@@ -60,4 +60,5 @@ xkb_symbols "common-keys" {
     // These should be camera focus and camera snapshot, but there are no better symbols available
     key <I536>  { [     XF86ContrastAdjust                                              ] };
     key <I774>  { [     XF86WebCam                                                      ] };
+    key <I361>  { [     Select                                                          ] };
 };
-- 
1.8.3-rc3

